### DISCUSSION TITLE
Retrieve logged in username in Script

### DISCUSSION THREAD

**Discussion by tux-mania** [netbox-community/netbox#11941] [Q&A] [answered]
Hello.

I need to retrieve the username of the logged in user for a specific use case in a Script.
How can I get this value please?

Thanks for your help.

**Comment by jeremystretch** [comment, marked as answer]
The user executing the script is accessible as an [attribute of the request object](https://docs.netbox.dev/en/stable/customization/custom-scripts/#accessing-request-data). Hope that helps!

**Reply by tux-mania** [reply]
Hi Jeremy! 
Thank you very much for the link! It definitely resolved my problem!
By the way, taking advantage of your answer to thank you and congrats you for this amazing project Netbox which makes daily work easier! :)
